FirstWebApp/pom.xml

```diff
@@ -46,6 +46,16 @@
             <version>9.0.45</version>
         </dependency>
 
+        <dependency>
```

Comment:
can also try to use gradle build system ?

Reply:
Yeah, I want to try that! Let me exhaust myself with Maven and I will move on to Gradle.